frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -79,8 +80,17 @@ const Sidebar = ({
 }: SidebarProps) => {
   const { t } = useTranslation();
   const { pathname } = useLocation();
+  const [ canUpgrade, setCanUpgrade ] = useState(false);
   const { activeSidebar, sidebarStatus, toggleSidebar } = useContext(AppContext);
 
+  useEffect(() => {
+    if (deviceIDs.length) {
+      getVersion(deviceIDs[0]).then(versionInfo => setCanUpgrade(versionInfo.canUpgrade));
```

Comment:
In the sidebar this only checks for the first deviceID, in settings tab it checks every deviceID.

If we want proper multi device support here, we could check every deviceID and add the dot if one of them canUpgrade.

Should update this? cc @benma

Reply:
Even though we support only one, it's easy to check all of them, right? So why not.

I think this code fails if a BitBox01 is connected, or the BitBox02 bootloader. You probably need to pass `devices` not `deviceIDs` to the sidebar, which contains the `TProductName`, which is `'bitbox' | 'bitbox02' | 'bitbox02-bootloader'`, so you should only call getVersion if the product name is `bitbox02`